source/use-cases/sensitive-data-encryption.txt

```diff
@@ -75,20 +75,61 @@ velit congue vel.
-Fusce ullamcorper neque eu libero aliquam, sed placerat sapien cursus.
-Vestibulum maximus magna sed velit feugiat tempor.
+Read and write operations on encrypted fields use the same methods and have the capabilities of traditional MongoDB read and write operations with the added security of field-level encryption.
 
-Ut accumsan pellentesque metus, at porta felis dapibus vulputate. Aenean
-malesuada interdum libero in feugiat. Fusce pharetra massa eget leo
-varius, et gravida massa tempus.
+This section continues the sample guide of the MedcoMD Management System. Now that their schemas are defined, MedcoMD engineers are tasked with creating a system that allows doctors, who have access to the FLE Enabled client, to read and write to documents in the patient’s collection without restriction. Receptionists, who have access to the non-FLE enabled client, have restricted access to documents in the patient’s collection that prevents them from reading and writing to sensitive fields. 
+By following this section, you learn the difference in output when using a secure client with FLE enabled, and a client without FLE enabled.
+A patient is inserted into the patient’s collection using the following code. 
 
-Ut mattis eros elit, sed ultrices purus lacinia sed. Curabitur tincidunt
-tellus eget neque porta dictum ac vel orci. Vestibulum ante ipsum primis
-in faucibus orci luctus et ultrices posuere cubilia Curae.
 
-Praesent feugiat arcu non risus elementum eleifend. Nullam sed iaculis
-erat.
+.. tabs::
+
+   tabs:
+
+     - id: java-jsonschema-generation
+       name: "Java"
+       content: |
+       
+        .. code-block:: java
+
+            public static void insertPatient(MongoCollection collection, String name, int ssn, String bloodType, ArrayList<Document> medicalRecords, int policyNumber, String provider) {
+                Document patient = new Document();
+                patient.put("name", name);
+                patient.put("ssn", ssn);
+                patient.put("bloodType", bloodType);
+                patient.put("medicalRecords", medicalRecords);
+                Document insurance = new Document();
+                insurance.put("policyNumber", policyNumber);
+                insurance.put("provider", provider);
+                patient.put("insurance", insurance);
+                collection.insertOne(patient);
+            }
+
+A doctor starts entering her new patient’s information in the Medco Management System. When she hits submit the above code is executed using the FLE enabled client. The new patient is inserted into the database, with the sensitive fields encrypted and looks like this:
+
+.. code::
+
+    Document{{_id=5d696fff112d8c439bae7068, name=Peter Parker, ssn=org.bson.types.Binary@afbb5147, bloodType=org.bson.types.Binary@713daeda, medicalRecords=org.bson.types.Binary@113f42c8, insurance=Document{{policyNumber=org.bson.types.Binary@806ab88d, provider=MaestCare}}}}
```

Comment:
- I think this would be more effective if we displayed a valid JSON interpretation of the document rather than this Java representation (essentially for the same reasons as the JSON schema section)
- Let's prettify this to split it across multiple lines. Horizontal scrolling code blocks kinda suck.

Reply:
In the latest commit, I've switched over to JSON (partially because a prettified java Document still doesn't look great and also because of Nick's first bullet).

Let me know what you think